ResearchKit/Common/ORKImageCaptureStep.h

```diff
@@ -65,4 +65,14 @@ ORK_CLASS_AVAILABLE
  */
 @property (nonatomic) UIEdgeInsets templateImageInsets;
 
+/**
+ Accessibility hint of the preview.
+ */
+@property (nonatomic, strong) NSString *accessibilityInstructions;
+
```

Comment:
I would suggest a more descriptive documentation of `accessibilityInstructions` since it is not part of the Accessibility Programming Guide for iOS. Include an example in the documentation to explicitly indicate how it should be used.